vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1391,6 +1417,7 @@ def _get_prompt_logprobs_dict(
     def _dummy_run(
         self,
         num_tokens: int,
+        initialize_attention_metadata: bool = False,
```

Comment:
nit: Maybe we can rename this for clarity?
```suggestion
        skip_attn: bool = True,
```

Reply:
renamed, also flipped it to False by default.